apps/extension/webpack.config.js

```diff
@@ -10,8 +10,8 @@ const RemovePlugin = require("remove-files-webpack-plugin");
 const packageJson = require("./package.json");
 const { getProcessEnv } = require("@namada/config/webpack.js");
 
-// Load .env from namada-interface:
-require("dotenv").config({ path: "../namada-interface/.env" });
+// Load .env from namadillo:
+require("dotenv").config({ path: "../namadillo/.env" });
```

Comment:
I know these are at least used for the default chain ID when you load the extension. I'm not sure about other places but I think at least wherever the chain configs are used, the `.env` file will still be relevant. I mainly fixed this because it was annoying me in development :smile: